pkgs/top-level/all-packages.nix

```diff
@@ -1798,6 +1798,8 @@ with pkgs;
 
   gh-markdown-preview = callPackage ../tools/misc/gh-markdown-preview { };
 
+  gh-screensaver = callPackage ../tools/misc/gh-screensaver { };
```

Comment:
Can you use the new by-name structure? https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#readme

Reply:
If I understand this correctly, I just need to put the package in the appropriate directory in `pkgs/by-name`, rename the file to `package.nix`, and remove the attribute in `all-packages.nix` and leave it at that? (This is the first  time I've heard of by-name structure.)